WalletWasabi.Documentation/ClientRelease/ReleaseNotesTemplate.md

```diff
@@ -0,0 +1,39 @@
+### _[Wasabi](https://WasabiWallet.io) is an easy to use, privacy-focused, open-source, non-custodial, Bitcoin wallet_
+
+# Download
+:window: [Windows](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.4/Wasabi-2.0.4.msi)
+:green_apple: [Apple M1/M2](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.4/Wasabi-2.0.4-arm64.dmg)
+:apple: [Apple Intel](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.4/Wasabi-2.0.4.dmg)
+:penguin: [Ubuntu / Debian](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.4/Wasabi-2.0.4.deb)
+:penguin: [Other Linux](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.4/Wasabi-2.0.4.tar.gz)
```

Comment:
```suggestion
:window: [Windows](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.msi)
:green_apple: [Apple M1/M2](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x-arm64.dmg)
:apple: [Apple Intel](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.dmg)
:penguin: [Ubuntu / Debian](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi2.0.x.deb)
:penguin: [Other Linux](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.tar.gz)
```

Reply:
fixed, you made a typo ;) Wasabi2.0.x.deb `-` is missing